en/getting_started/installation.md

```diff
@@ -9,58 +9,51 @@ including both [XML message definitions](../messages/README.md) and the GUI/comm
 
 ## Prerequisites
 
-The requirements for using the *MAVLink tools* are: 
+The requirements for using the *MAVLink generator* are: 
 
-* Python 3.3+ (recommended) or Python 2.7+
+* Python 3.3+
```

Comment:
You might as well write Python 3.6+
https://endoflife.date/python

Reply:
Fair enough!

Reply:
Thanks!